.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "cosim/import/basejump_stl"]
```

Comment:
Use the basejump submoduled in BlackParrot? Otherwise could get into a versioning issue